### PR TITLE
Fixes overflow for long calendar names

### PR DESCRIPTION
**Pull request by zomars**
## What does this PR do?

- Adds title attribute for overflowed text
- Fixes overflowed text for long calendar names 

Before:

image
 

After:

image

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- [ ] Add a very long calendar name and see the primary calendar selector

### CODE REVIEW

**Review by sean-brydon — COMMENTED** [review]
LGTM :)

**Review by sean-brydon — APPROVED** [review]
LGTM :)

**Review by PeerRich — APPROVED** [review]

File: apps/web/components/DestinationCalendarSelector.tsx [hunk 1]
```diff
@@ -53,11 +53,14 @@ const DestinationCalendarSelector = ({
...
+    <div className="relative" title={`${t("select_destination_calendar")}: ${selectedOption?.label || ""}`}>
       {/* There's no easy way to customize the displayed value for a Select, so we fake it. */}
       {!hidePlaceholder && (
-        <div className="pointer-events-none absolute z-10">
-          <Button size="sm" color="secondary" className="m-[1px] rounded-sm border-transparent">
+        <div className="pointer-events-none absolute z-10 w-full">
+          <Button
+            size="sm"
+            color="secondary"
+            className="m-[1px] w-[calc(100%_-_40px)] overflow-hidden overflow-ellipsis whitespace-nowrap rounded-sm border-none leading-5">
```
Comment (PeerRich) [inline apps/web/components/DestinationCalendarSelector.tsx:63]: `w-[calc(100%_-_40px)]` i didn't know you could do that. that's neat

Comment (zomars) [reply apps/web/components/DestinationCalendarSelector.tsx:63]: Arbitrary values are a game changer in v3: https://tailwindcss.com/docs/adding-custom-styles#using-arbitrary-values

**Review by leog — APPROVED** [review]
LGTM
Also, I was meaning to say this a long time now: Is there a way we can stop getting the `ESLint Report Analysis` warnings, or perhaps find solutions for those checks? I find it very distracting when it comes to reviewing a PR.

**Comment by zomars** [comment]
> LGTM
> Also, I was meaning to say this a long time now: Is there a way we can stop getting the `ESLint Report Analysis` warnings, or perhaps find solutions for those checks? I find it very distracting when it comes to reviewing a PR.

Feel free to tackle those in a PR ;)

**Pull request merged**